mergify_cli/__init__.py

```diff
@@ -844,6 +854,13 @@ async def parse_args(args: typing.MutableSequence[str]) -> argparse.Namespace:
         help="Branch prefix used to create stacked PR. "
         "Default fetched from git config if added with `git config --add mergify-cli.stack-branch-prefix some-prefix`",
     )
+    stack_parser.add_argument(
+        "--rework",
+        "-r",
+        action="store_true",
+        default=False,
+        help="Rework the order or the content of the stack",
+    )
```

Comment:
This sounds like it should be a command and not an option?

Reply:
I hesitated. What about `mergify restack`?

Reply:
That or we keep `stack` as a prefix for not the confused without future commands for other products.

Reply:
you mean something like `mergify stack rework`?

Reply:
Yes

Reply:
Also not sure "rework" is the best wording.

Reply:
Maybe `mergify stack edit`